content/blog/developer-experience-detailed.md

```diff
@@ -0,0 +1,93 @@
+---
+date: 2024-03-15
+title: 'Serverless Workflows: an Automated Developer Experience Step-by-Step'
+---
+In this blog, we'll guide you through the journey from a software template to bootstrapping the workflow development, building, packaging, releasing, and deploying it on a cluster. If you need a high-level explanation or want to dive into the architecture of the solution, check out our previous blog. You can also watch a detailed demonstration of the content covered in this post in this [recording](https://www.youtube.com/watch?v=G6wnRHjvhv0).
```

Comment:
Should there be a link here to the previous blog?

Reply:
+1 for linking both the blog post and the recording.

Reply:
added.